qt/app.py

```diff
@@ -1727,6 +1736,23 @@ def suspendMouseButtonNavigation(self):
         yield
         self.setMouseButtonNavigation()
 
+    # |-------|
+    # | Slots |
+    # |-------|
+    def slot_setup_language(self):
+        dlg = LanguageDialog(
+            used_language_code=self.config.language_used,
+            configured_language_code=self.config.language())
+
+        dlg.exec()
+
+        if dlg.result() == 1:
+            self.config.setLanguage(dlg.language_code)
+
+            # How to send notify message (systray bubble)?
```

Comment:
It is possible to modify the language used by gettext and Qt on the fly without restarting the whole application. But BITs current code structure do not allow it. So users have to restart BIT as most other GUI applications to my knowledge realizing it.

Here I would like to inform the users about that fact. I simple systray bubble I would say. How can we do this with BIT?

Reply:
I was looking in #1156 and found about `NotifyPlugin` class. But I can not see where it is used. It seems to be unused.
I also tried something like `self.config.PLUGIN_MANAGER.error("foo")` but without effect.

I don't understand how BIT produce the systray bubbles/balloons.

Reply:
I could find `common/pluginmanager.py` and the class `PluginManager` in there.
I don't understand that approach. Why do I need a "plugin" to send notify messages? And all this happens without `logger.py`.
If I would understand the intention behind that approach I could come up with a solution/improvement.

This smells like a lot of refactoring and redesign work. So I'll stop at this point.

The question is how should we inform the user about the fact that the language settings taking effect only after restarting BIT? IMHO a message box would be a pragmatic solution.

Reply:
![Peek 2023-08-11 12-26](https://github.com/bit-team/backintime/assets/11861496/1e64af6a-9742-45af-a3ff-80ed601699c9)

This is how it looks like with a info box that shows up only when the language was modfied.